app/experimenter/nimbus-ui/src/components/PageResults/TableVisualizationRow/index.tsx

```diff
@@ -170,6 +170,17 @@ const percentField = (
   );
 };
 
+const formattedAnalysisPointComparator = (
+  a: FormattedAnalysisPoint,
+  b: FormattedAnalysisPoint,
+) => {
+  if (!a.window_index || !b.window_index) {
+    return 0;
+  }
+  if (a.window_index < b.window_index) return -1;
+  if (a.window_index > b.window_index) return 1;
```

Comment:
Do we need a catch-all `return 0;` so it doesn't return null/undefined?
Or something like `return b.window_index - a.window_index;` which might cover both `if..if..else` statements.

Reply:
good catch, thank you, updated